test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsSqsTests.cs

```diff
@@ -0,0 +1,169 @@
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+
+            // ReceiveMessagesAndDeleteMessageBatch
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // DeleteQueue
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+
+#endif
+            // CreateSqsQueueAsync
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySQSQueue"),
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySQSQueue2"),
+
+            // SendMessagesWithInjectedHeadersAsync
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // SendMessagesWithoutInjectedHeadersAsync
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // SendMessageAsync
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("SendMessage"),
+
+            // ReceiveMessageAndDeleteMessageAsync
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+
+            // SendMessageBatchAsync
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+
+            // ReceiveMessagesAndDeleteMessageBatchAsync
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // DeleteQueueAsync
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+        };
+
+        public AwsSqsTests(ITestOutputHelper output)
+            : base("Aws.Sqs", output)
+        {
+        }
+
+        [Theory]
+        [MemberData(nameof(PackageVersions.AwsSqs), MemberType = typeof(PackageVersions))]
+        [Trait("Category", "EndToEnd")]
+        public void SubmitsTraces(string packageVersion)
+        {
+            SetCallTargetSettings(enableCallTarget: true, enableMethodInlining: true);
```

Comment:
This method doesn't existing any more in master! Looks like you need a rebase :)
```suggestion
            SetCallTargetSettings(enableCallTarget: true);
```

Reply:
Thanks, I finished rebasing and fixed this method invocation in 23bf71d649949821743706ee64a69c287d3fe9b9